config/environments/development.rb

```diff
@@ -47,6 +47,7 @@
 
   config.hosts << 'api.lago.dev'
   config.hosts << 'api'
+  config.hosts << "lago.ngrok.dev"
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)